recipes-tools/prometheus/prometheus.bb

```diff
@@ -28,6 +28,7 @@ do_install() {
 
     install -d ${D}${sysconfdir}/prometheus
     install -m 0644 ${WORKDIR}/prometheus.yml.mustache ${D}${sysconfdir}/prometheus/
+    install -m 0640 ${WORKDIR}/prometheus.yml.mustache ${D}${sysconfdir}/prometheus/prometheus.yml
```

Comment:
what was the reason for the duplication here?

Reply:
The rendered config (for some reason) is owned by `root:root` with permissions set to 640 making it unreadable by the `prometheus` user. This line pre-creates the file with the correct permissions and the mode.